modules/kube-loadbalancer-sg/main.tf

```diff
@@ -0,0 +1,26 @@
+/**
+ * ## Kube Load Balancer Security Group
+ *
+ * Define a security group for kube load balancer
+ *
+ */
+
+module "kube-load-balancer-sg" {
```

Comment:
I'm not sure if we should rename this to `main`, `security-group`, `sg`, or `lb-sg`, but I think one of these will be better (as seen through the Terraform state).